geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/CreateJndiBindingCommandDUnitTest.java

```diff
@@ -69,8 +73,10 @@ public void testCreateJndiBinding() throws Exception {
 
     // verify cluster config is updated
     locator.invoke(() -> {
+      InternalLocator locator = ClusterStartupRule.getLocator();
```

Comment:
change "locator" to "internalLocator". The outer scope has its own "locator" so better not to have this name shadowed.